### PR TITLE
Responses to the rest of the reviews

### LINKED ISSUE #43: Comments from Wilder
comments from @awohns

- Page 5, Example 1 Ancestry proportions: this seems quite similar, if not equivalent to the Genomic Descent statistic I defined in https://www.tandfonline.com/doi/full/10.1080/03014460.2019.1623912.

- Page 13 caption to figure 5: “tree sequences inferred by Relate”, not quite tree sequences though, right?

### PR DESCRIPTION
**Pull request by petrelharp**
I've made some other edits for reviewer comments - note the genotype matrix in Figure 2 - but among other things, there's edits in response to #43.

### CODE REVIEW

**Comment by petrelharp** [comment]
Ok, this is a full draft of the final response. I didn't put the node statistics first - maybe I should have? - and maybe the general message of "a general way to summarize trees is by propagating weights up them and applying a summary function" has been some what diluted... but see what you think?

File: treestats_paper.tex [hunk 1]
```diff
@@ -224,9 +222,100 @@ \section*{Framework and statistics}
...
+However, since the genetic variation represented in a genotype matrix
+was generated by mutations inherited down genealogical trees,
+it is often informative to think about these statistics in terms of these trees.
+Most population genetic summary statistics can be thought of in terms of the \emph{allele frequencies},
+i.e., what proportion of the sampled genotypes carry each allele.
+The frequency of any particular allele
+can easily be found from the location of the mutation(s) that produced it
+on the marginal genealogical tree,
+by counting the number of sampled genotypes below it in the tree.
+If many mutations have occurred on a single genealogical tree,
```
[the remark this answers is not in the document]
Comment (petrelharp) [reply treestats_paper.tex:240]: Maybe just deleting the second half? I kinda want to remind people about the other direction, genotypes -> trees.

**Comment by petrelharp** [comment]
I've done something about all of these (good) comments. See what you think?

File: treestats_paper.tex (hunk 1 above)
[the remark this answers is not in the document]
Comment (jeromekelleher) [reply treestats_paper.tex:240]: Sure, sounds good.

**Comment by jeromekelleher** [comment]
Looks great, thanks!

**Pull request merged**